src/extension/ip_extraction.c

```diff
@@ -520,34 +524,44 @@ static bool _is_private_v6(const struct in6_addr *nonnull addr)
     static const struct {
         union {
             struct in6_addr base;
-            unsigned __int128 base_i;
+            uint64_t base_i[2];
```

Comment:
Similarly to my comment below, we can use `__int128`

Reply:
Done